scenario/LiquidationBotScenario.ts

```diff
@@ -621,103 +669,129 @@ scenario(
   }
 );
 
-scenario(
-  `LiquidationBot > reverts when price slippage is too high`,
-  {
-    upgrade: {
-      targetReserves: exp(20_000, 18)
+{
+  const baseTokenBalances = {
+    mainnet: {
+      usdc: 2250000,
+      weth: 20
     },
-    filter: async (ctx) => matchesDeployment(ctx, [{network: 'mainnet'}]), // XXX enable for Polygon
-    tokenBalances: {
-      $comet: { $base: 10000 },
+    arbitrum: {
+      usdc: 10000000
+    }
+  };
+  const assetAmounts = {
+    mainnet: {
+      usdc: ' == 5000', // COMP
+      weth: ' == 10000', // CB_ETH
     },
-    cometBalances: {
-      albert: {
-        $asset0: ' == 10000',
+    arbitrum: {
+      usdc: ' == 2000000', // ARB
+    }
+  };
+
+  scenario(
+    `LiquidationBot > reverts when price slippage is too high`,
+    {
+      upgrade: {
+        targetReserves: exp(20_000, 18)
       },
-      betty: { $base: 1000 },
+      filter: async (ctx) => matchesDeployment(ctx, [{network: 'mainnet'}]),
```

Comment:
Was it intentional to remove `arbitrum` from this list? If so, we could remove the `arbitrum` key in the `baseTokenBalances` and `assetAmounts` mapping.

Reply:
yeah, it's intentional. we can not borrow a large enough sum from the flash loan pool to create enough slippage to make this test revert.

deleted the `arbitrum` keys from `baseTokenBalances` and `assetAmounts`